README.md

```diff
@@ -1,12 +1,20 @@
 # reveal.js 
 
-A framework for easily creating beautiful presentations using HTML. [Check out the live demo](http://revealjs.com/).
+A framework for easily creating beautiful web-based presentations. [Check out the live demo](http://revealjs.com/).
 
->__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 to continue active development and feature contributions to reveal.js. Upstream This fork [is published](https://npmjs.com/package/@h3/reveal.js) on npmjs under the `@h3` scope. See [CONTRIBUTING](CONTRIBUTING.md) for info on how to join [HackerHappyHour](https://github.com/HackerHappyHour) and contribute!
+>__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 
+to continue active development and feature contributions to reveal.js. Upstream of this fork 
```

Comment:
Not sure how I got the 'Upstream' word mixed in there.. it doesn't make sense when you read the full sentence.
```suggestion
to continue active development and feature contributions to reveal.js. This fork 
```